python/tox.ini

```diff
@@ -15,33 +15,37 @@
 #
 
 [tox]
-envlist = py36, py37, py38, py39, flake8
+envlist = py36, py37, py38, py39, flake8, pylint
 
 [gh-actions]
 python =
     3.6: py36
     3.7: py37
-    3.8: py38, flake8
+    3.8: py38, flake8, pylint
```

Comment:
just wondering: why do we run style tests on 3.8 and not 3.9? I'd guess going with the latest version for style-related stuff would be preferable from the general code evolution perspective?

Reply:
good question, from my side, I have no idea 😄.